tests/unit/qmlparsertestcase.py

```diff
@@ -141,5 +147,6 @@ def test_keyword_arguments(self):
         qmlparser.parse(lexer.tokens, qmlclass)
 
         functions = qmlclass.get_functions()
+        self.assertEqual(len(functions), 1)        
```

Comment:
All these assert changes make sense, but they have nothing to do with the relative import problem: they should be done in a separate pull request.

Reply:
Agreed. I will create a separate pull request.